src/mixins/dropdown.js

```diff
@@ -350,7 +350,9 @@ export default {
       if (this.$el.contains(evt.relatedTarget)) {
         return
       }
-      this.visible = false
+      setTimeout(() => {
+        this.visible = false
+      })
```

Comment:
Would a `this.$nextTick(() => { this.visible = false })` work as well, or would that be too much of a delay?

Reply:
Went with ’setTimeout()’ because ’$nextTick()’ wasn't helping here.

Reply:
OK cool.